### ISSUE TITLE
Network Changes, Phlex completely bombing

### ISSUE THREAD

**Issue by Cyberchaos** [d8ahazard/Phlex#120]
Hey,

My struggle with Phlex continues. So I was working on my network today, had to reboot my router about 70 times which reassigns IPs, both internal and external. Anyway, Phlex is just generally being slow and shitty, so I create a new folder in htdocs on XAMPP called Phlex2, just to see if it's any snappier. No such luck, also not working.

Go back to my original installation, now it's saying API token invalid, not that I changed that API token at all, or linked another Phlex instance or anything.

Long story short, fuck it, let's try to start from scratch. Unlink Google account on Home, delete phlex folders on XAMPP, copy a fresh version in, start up XAMPP aaaand same problem.

How do I fix this mess?

**Comment by XmemphistoX** [comment]
1. You should set static IPs and or DHCP reservations on your Plex host and Phlex host.  If you reboot your router, the IPs will stick.

2. Setup a DNS name for your WAN network and register Phlex to that WAN name (Not IP address).  That way if your router/modem reboots and you get a new WAN IP, your DNS name will stay the same.

Phlex being "slow and shitty" isn't much of a diagnosis.  You'll have to give a bit more detail.  Its like someone bringing in a car to a mechanics saying my car is slow and shitty, fix it - without any context.

**Comment by Cyberchaos** [comment]
Hey,

I'm aware it's not a great diagnosis, hernce the latter part. I'm now trying to "reinstall" Phlex to start from scratch, but index.php just takes forever to load - I'm talking times out half the time. I'm running on XAMPP and have done all the restarts - router now setup as necessary to stick to the given IP, so that should be fine, but index just won't load.

The only time it remotely does is when I open a new browser/incognito into the localhost Phlex IP, but then it doesn't log in, and, if it does, doesn't update my public IP or find my playback server (for which I've done all the standard "erase dlist and PlexServer lines from config" fix to no avail).

So I'm a bit lost on troubleshooting really. Obviously a ton of Phlex is handled calling to some outside host, and part of me thinks this is why I'm having all these issues. Internet, Plex and everything else seems fine, it's just Phlex which is a mess.
